whitepaper/whitepaper.md

```diff
@@ -33,6 +37,11 @@ Photo by Pontus Wellgraf on Unsplash
 <div style="page-break-after: always; visibility: hidden"><!-- \pagebreak --></div>
 
 - [1. Abstract](#1-abstract)
+  - [1.1 Terminology of Blockchain Interoperability](#11-terminology-of-blockchain-interoperability)
```

Comment:
I am afraid proposed contents are not abstract. I might suggest you to add new chapter titled like: "Introduction".
The Introduction should cover what we will deliver with Cactus as integrated service builder.